src/components/BottomBarTabView.vue

```diff
@@ -84,10 +72,14 @@ export default {
       this.$emit('bottomBarTabToggle', courseObj);
     },
 
-    deleteBottomTab(subject, number) {
-      for (let i = 0; i < this.bottomCourses.length; i += 1) {
-        if (this.bottomCourses[i].subject === subject && this.bottomCourses[i].number === number) {
+    deleteBottomTab(courseObj) {
+      let focusedCourse = this.bottomCourses[this.bottomCourseFocus];
+
+      let i = 0;
+      for (; i < this.bottomCourses.length; i += 1) {
```

Comment:
Any reason the let is not in the first part of the for loop here? Didn't know this was possible haha.

Reply:
It's so I can use `i` in line 101 further down in the function:
```
this.bottomBarTabToggle(this.bottomCourses[i]);
```
Although if it makes more sense or is more readable, I can put `i` back in the for loop and use another variable.

Reply:
Nope this is fine! Just took me a second to figure it out 😄